.cirrus.yml

```
@@ -0,0 +1,352 @@
+
+task:
+  name: 'macOS 13 native arm64 [gui, sqlite only] [no depends]'
+  macos_instance:
+    # Use latest image, but hardcode version to avoid silent upgrades (and breaks)
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:14.3.1  # https://cirrus-ci.org/guide/macOS
+  << : *BASE_TEMPLATE
+  check_clang_script:
+    - clang --version
+  brew_install_script:
+    - brew install boost libevent qt@5 miniupnpc libnatpmp ccache zeromq qrencode libtool automake gnu-getopt
+  << : *MAIN_TEMPLATE
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)